CHANGELOG.md

```diff
@@ -28,6 +28,8 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Dimitris Koutsogiorgas](https://github.com/dnkoutso)
   [#8585](https://github.com/CocoaPods/CocoaPods/pull/8585)
 
+* Fix embedding static frameworks in extensions while using `use_frameworks!`  
+  [Martin Fiebig](https://github.com/mfiebig)
```

Comment:
can you add `[#8798](https://github.com/CocoaPods/CocoaPods/pull/8798)`?

Reply:
I changed that, thanks!